code/modules/vehicles/mecha/mech_melee_attack.dm

```diff
@@ -42,7 +42,13 @@
 			return 0
 	mecha_attacker.visible_message(span_danger("[mecha_attacker] hits [src]!"), span_danger("You hit [src]!"), null, COMBAT_MESSAGE_RANGE)
 	..()
-	return take_damage(mecha_attacker.force * 3, mecha_attacker.damtype, "melee", FALSE, get_dir(src, mecha_attacker)) // multiplied by 3 so we can hit objs hard but not be overpowered against mobs.
+	return take_damage(mecha_attacker.force * get_obj_mecha_damage_multiplier(), mecha_attacker.damtype, "melee", FALSE, get_dir(src, mecha_attacker)) // multiplied by 3 so we can hit objs hard but not be overpowered against mobs.
```

Comment:
Move this comment to the new proc. 
`return 3 // multiplied by 3 by default so we can hit objs hard but not be overpowered against mobs.`